.browserlistrc

```diff
@@ -0,0 +1,3 @@
+> 1%
```

Comment:
Please review the supported web browsers scope.
https://docs.processmaker.com/using/web-browser-requirements
https://browsersl.ist/#q=%3E++1%25%0Alast+2+versions%0Anot+dead